content/800-data-platform/250-environments/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Environments'
+metaTitle: 'Environments'
+metaDescription: 'In a project in the Prisma Data Platform, you can create multiple environments to associate a Prisma schema on a repository branch with a database and, thus, separate database development activities with JavaScript/TypeScript and Prisma ORM.'
+tocDepth: 3
+---
+
+<TopBlock>
+
+For any project in the Prisma Data Platform, you can create multiple environments and associate each environment with a different database, Data Proxy location, and a different branch of your GitHub repository.
```

Comment:
```suggestion
For any project in the Prisma Data Platform, you can create multiple environments. You can associate each environment with a different database, Data Proxy location, and branch of your GitHub repository.
```

Reply:
Should we link to this intro on the first mention of "environment" on each of the other pages?